script.js

```diff
@@ -0,0 +1,197 @@
+let todoList = [];
+let doneList = [];
+
+// 입력된 값을 todo list에 추가
+const addTodoItem = () => {
+  const todoContent = document.querySelector(".todo-input").value;
+  if (todoContent) paintTodoItem(todoContent);
+};
```

Comment:
```suggestion
const addTodoItem = (event) => {
  const todoContent = event.target.value;
  if (todoContent) paintTodoItem(todoContent);
};
```
이벤트 핸들러 함수는 이벤트 객채에서 값을 가져오는 것이 일반적입니다.

Reply:
그렇군요! 다음부터 참고해서 코드짜도록 하겠습니다!!